src/js/tracks/video-track-enums.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file video-track-enums.js
+ */
+
+/**
+ * https://html.spec.whatwg.org/multipage/embedded-content.html#dom-videotrack-kind
+ *
+ * enum VideoTrackKind {
+ *   "alternative",
+ *   "captions",
+ *   "main",
+ *   "sign",
+ *   "subtitles",
+ *   "commentary",
+ *   "",
+ * };
+ */
+const VideoTrackKind = {
```

Comment:
Ah, interesting, only TextTrack has actual defined `enum`: https://html.spec.whatwg.org/multipage/embedded-content.html#texttrackkind...

Specifically, we can do something like:
```js
let kind = AudioTrackKind[options.kind] || "";
```
I was thinking we could also technically add a `_default` property to the `Kind` object (with the prefixed `_` to make sure it will never conflict with the spec) and then we could do:
```js
let kind = AudioTrackKind[options.kind] || AudioTrackKind._default;
```

So, yeah, it's even less of a real enum than for text tracks, but I think having the object in a single file exported under two names will reduce code duplication and allow other code to be a bit neater.